tests/test_widgets.py

```diff
@@ -214,3 +214,53 @@ def test_unhashable_choice_data():
     assert combo.choices == ([1, 2, 3], [1, 2, 5])
     combo.choices = ("x", "y", "z")
     assert combo.choices == ("x", "y", "z")
+
+
+def test_bound_values():
+    @magicgui(x={"bind": 10})
+    def f(x: int = 5):
+        return x
+
+    assert not f.x.visible
+    assert f() == 10
+    f.x.unbind()
+    assert f() == 5
```

Comment:
Is it worth adding one more test with the `visible=True` case. Did I get the syntax right?

```python
def test_bound_values_visible():
    @magicgui(x={"bind": 10, "visible": True})
    def f(x: int = 5):
        return x

    assert f.x.visible
    assert f() == 10
    f.x.unbind()
    assert f() == 5
```

Reply:
yes indeed!  will do

Reply:
added

Reply:
thanks!